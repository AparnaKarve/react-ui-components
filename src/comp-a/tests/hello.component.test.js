import React from 'react';
import { HelloCmp } from '../some-cmp/hello.component';
import renderer from 'react-test-renderer';

test('Some dummy test', () => {
  const component = renderer.create(
    <HelloCmp></HelloCmp>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
