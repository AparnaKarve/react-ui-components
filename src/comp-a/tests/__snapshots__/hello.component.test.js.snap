// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some dummy test 1`] = `
<div>
  Hello
</div>
`;
